### ISSUE TITLE
Axis Alignment

### ISSUE THREAD

**Issue by alfredgu001324** [yuantianyuan01/StreamMapNet#20]
Dear author,

I noticed that you have addressed several times already, but I still find it a bit unclear. I notice that in your work bev_h is 50 and bev_w is 100. I assume this means that the vehicle is facing the positive X axis. But what about the positive Y axis? Does that mean the leftward direction of the vehicle? Or after you flip the Y axis, actually downward means leftward?

**Comment by yuantianyuan01** [comment]
Yes, I think you've almost got it. The positive X-axis points rightward on BEV feature space (and the vehicle is facing positive X-axis too). The positive Y-axis points upward on BEV, which means the lefthand side of the vehicle.

**Comment by KSonPham** [comment]
@yuantianyuan01 Can you explain why there is a minus in this part?
https://github.com/yuantianyuan01/StreamMapNet/blob/000ce22f2fae6a1798a57471f820b8232ae76a74/plugin/models/mapers/StreamMapNet.py#L135
may be there smt to do with this line but I don't understand why did you do that:
https://github.com/yuantianyuan01/StreamMapNet/blob/000ce22f2fae6a1798a57471f820b8232ae76a74/plugin/models/mapers/StreamMapNet.py#L66
